hack/ci/e2e.sh

```diff
@@ -187,9 +187,29 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
-    # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    # ginkgo regexes, use CI k/k presubmit regex by default
+    SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+    FOCUS="${FOCUS:-""}"
+
+    # skip known issues
+    # [sig-node] RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]
+    # https://github.com/kubernetes/kubernetes/issues/80098
+    SKIP="RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler|${SKIP}"
+    # [sig-storage] In-tree Volumes [Driver: nfs]
+    # [sig-storage] PersistentVolumes NFS
+    # TODO: kind nodes can't mount nfs volumes: install apt install -y nfs-common in the base image, ...??
+    SKIP="In-tree Volumes \\[Driver: nfs\\]|PersistentVolumes NFS|${SKIP}"
+    # [sig-storage] PersistentVolumes-local
+    # fails because sh: 1: sudo: not found, TODO check if we can detect sudo test/e2e/storage/utils/local.go:
+    SKIP="PersistentVolumes-local|${SKIP}"
+    # [sig-auth] PodSecurityPolicy
+    SKIP="PodSecurityPolicy|${SKIP}"
+    # [sig-network] Network should set TCP CLOSE_WAIT timeout
+    # Use ssh to login to the kind nodes TODO: change to docker exec commands, install ssh server, ...
+    SKIP="Network should set TCP CLOSE_WAIT timeout|${SKIP}"
+    # [sig-cli] Kubectl client [k8s.io] Simple pod should support exec through an HTTP proxy 
+    SKIP="Simple pod should support exec through an HTTP proxy|${SKIP}"
```

Comment:
🤷‍♂ I'm not able to aggregate all the tests to create the SKIP regex
/help @BenTheElder @neolit123

Reply:
IIRC with ginkgo you cannot use spaces and need to use . Instead.

Reply:
I can use a regex like that with spaces if I execute the e2e binary from the command line, will try the dots , thanks